### PR TITLE
Features/cdh sandbox

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/quickstarts/installation/cdh_sandbox-adlsg2_ld.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+What this guide will cover:
+
+- Installing WANdisco Fusion and a CDH Sandbox using the [docker-compose](https://docs.docker.com/compose/) tool.
+- Integrating WANdisco Fusion with ADLS Gen2 storage.
+- Live replication of sample data.
+
+If you would like to try something different with the CDH Sandbox, see:
+
+* [Migration of data to ADLS Gen2](./cdh_sandbox-adlsg2_lm.md)
+* [Live replication of data/metadata to Databricks](./cdh_sandbox_lhv_client-adlsg2_lan.md)
```
Comment (catrionabarker) [inline docs/quickstarts/installation/cdh_sandbox-adlsg2_ld.md:18]: I think Hive needs mentioning here (and in the already live HDP version)

File: docs/quickstarts/installation/cdh_sandbox-s3_lm.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+   Enter your email address and choose a password you will remember.
+
+1. Click on the **Settings** cog for the **s3** storage, and fill in the details for your AWS S3 bucket. See the [Info you will require](#info-you-will-require) section for reference.
+
+1. Click **Apply Configuration** and wait for this to complete.
+
+## Migration
+
+Follow the steps below to demonstrate migration of HCFS data from the CDH Sandbox to the AWS S3 bucket.
```
Comment (catrionabarker) [inline docs/quickstarts/installation/cdh_sandbox-s3_lm.md:106]: your AWS S3 bucket

File: docs/quickstarts/installation/cdh_sandbox_lhv_client-adlsg2_lan.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+Use this quickstart if you want to configure Fusion to replicate from a non-kerberized Cloudera (CDH) Sandbox to an Azure Databricks cluster.
+
+This uses the [WANdisco LiveAnalytics](https://wandisco.com/products/live-analytics) solution, comprising both the Fusion Plugin for Databricks Delta Lake and Live Hive.
+
+What this guide will cover:
+
+- Installing WANdisco Fusion and a CDH Sandbox using the [docker-compose](https://docs.docker.com/compose/) tool.
+- Integrating WANdisco Fusion with Azure Databricks.
+- Live replication of sample data and metadata.
```
Comment (catrionabarker) [inline docs/quickstarts/installation/cdh_sandbox_lhv_client-adlsg2_lan.md:15]: Do we need to mention Hive?

File: docs/quickstarts/installation/hdp_sandbox-adlsg2_ld.md [hunk 4]
```diff
@@ -71,6 +71,12 @@ Log in to your VM prior to starting these steps.
    * Second zone type = `adls2`
    * Second zone name = _press enter for the default value_
 
+1. Enter the CDH Sandbox zone details:
```
Comment (catrionabarker) [inline docs/quickstarts/installation/hdp_sandbox-adlsg2_ld.md:74]: Doc name is hdp sandbox - I'm confused!

File: docs/quickstarts/installation/hdp_sandbox-adlsg2_lm.md [hunk 5]
```diff
@@ -71,6 +71,12 @@ Log in to your VM prior to starting these steps.
    * Second zone type = `adls2`
    * Second zone name = _press enter for the default value_
 
+1. Enter the CDH Sandbox zone details:
```
Comment (catrionabarker) [inline docs/quickstarts/installation/hdp_sandbox-adlsg2_lm.md:74]: Doc name is hdp sandbox - I'm confused!

File: website/sidebars.json [hunk 6]
```diff
@@ -45,6 +45,11 @@
     "Installation": [
```
Comment (catrionabarker) [inline website/sidebars.json:45]: We should probably comment all of these changes out for now - or at least be prepared to before pushing live if UAT haven't looked in time

File: docs/quickstarts/installation/hdp_sandbox-adlsg2_ld.md (hunk 4 above)
Comment (benradstone) [reply docs/quickstarts/installation/hdp_sandbox-adlsg2_ld.md:74]: Good spot, this was a new additional step for the Sandbox to custom distribution option, and I will have likely copied and pasted from the CDH Sandbox doc without thinking.

File: website/sidebars.json (hunk 6 above)
Comment (benradstone) [reply website/sidebars.json]: I've removed them for now: https://github.com/WANdisco/wandisco-documentation/pull/99/commits/c77f1a29251df778362e46b845d5eb08ec6c1e3d

**Pull request merged**